README.md

```diff
@@ -157,10 +159,12 @@ export default App;
 You'll notice we also removed the `user` prop from these components, since we'll
 be accessing that data via context instead.
 
+We've also included our `Header` and `Profile` components as children of our UserProvider.
```

Comment:
We wrap all lines at 80 characters. The Rewrap VS Code extension is helpful for this